pkg/scheduler/scheduler.go

```diff
@@ -123,7 +124,15 @@ func (pc *Scheduler) loadSchedulerConf() {
 	klog.V(4).Infof("Start loadSchedulerConf ...")
 	defer func() {
 		actions, plugins := pc.getSchedulerConf()
-		klog.V(4).Infof("Successfully loaded scheduler conf, actions: %v, plugins: %v", actions, plugins)
+		confBytes, err := yaml.Marshal(map[string]interface{}{
```

Comment:
let's make it simple. Do not think it's a good idea to expose all parameters in log for user.

Reply:
done. just print action, plugin name